packages/browser/src/transports/base.ts

```diff
@@ -15,6 +15,7 @@ const CATEGORY_MAPPING: {
   event: 'error',
   transaction: 'transaction',
   session: 'session',
+  sessions: 'session',
```

Comment:
Is this really necessary ? I feel like SessionAggregates do not touch on browser at all